RIPS/rip-xxxx.md

```
@@ -0,0 +1,376 @@
+
+## Backwards Compatibility
+
+This RIP introduces backwards incompatible changes to the block derivation and verification of an L2.
+These changes are purely additive and do not break anything related to current user activity and experience, but they do
+require changes to the L2's L1->L2 derivation process and the verification logic in their output settlement contract(s).
+
+## Test Cases
+
+N/A 
+
+##  Reference Implementation
```

Markdownlint (0.37.0)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)